src/test/java/com/alipay/remoting/rpc/exception/ExceptionTest.java

```diff
@@ -341,4 +341,29 @@ public Executor getExecutor() {
         latch.await();
         Assert.assertEquals(InvokeServerException.class, ret.get(0).getClass());
     }
+
+    @Test
+    public void testGetBizClassLoaderException1() {
+        server.registerUserProcessor(new SimpleServerUserProcessor() {
+            @Override
+            public ClassLoader getBizClassLoader() {
+                throw new RuntimeException("getBizClassLoader fail.");
+            }
+        });
+
+        RequestBody req = new RequestBody(4, "hello world");
+        try {
+            client.invokeSync(addr, req, 3000);
```

Comment:
Add ` Assert.fail();`  after this line to indicate that  `client.invokeSync` must throw an excpetion

Reply:
done